fdbserver/MasterProxyServer.actor.cpp

```diff
@@ -819,7 +859,6 @@ ACTOR Future<Void> commitBatch(
 		if(firstStateMutations) {
 			ASSERT(committed[t] == ConflictBatch::TransactionCommitted);
 			firstStateMutations = false;
-			forceRecovery = false;
```

Comment:
Can you give a quick explanation about what this change does?

Reply:
This is to remove dead code. At line 853, if `forceRecovery` is false, we enter the `for` loop. Otherwise, this loop is skipped.

Reply:
It looks like line 857 calls `applyMetadataMutations` with a pointer to `forceRecovery`, in which case it could conceivably be modified.

Reply:
You are right! I reverted this change.